engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-packages.html

```diff
@@ -25,8 +25,13 @@ <h3 translate>Packages in {{ repository.name }}</h3>
   </div>
 
   <div data-block="actions">
+    <div bst-modal="removeContent()">
+      <div data-block="modal-header" translate-plural>Remove {{ detailsTable.numSelected  }} Packages?</div>
+      <div data-block="modal-body" translate-plural>Are you sure you want to remove the {{ detailsTable.numSelected }} packages selected?</div>
```

Comment:
Is this the right way to use translate-plural?  All the other examples in the code are quite a bit more verbose and specific.